build.gradle

```diff
@@ -266,6 +266,7 @@ project('build-info-extractor') {
     apply plugin: 'java-test-fixtures'
     description = 'JFrog Build-Info Extractor'
     dependencies {
+        implementation 'org.eclipse.jgit:org.eclipse.jgit:5.1.16.202106041830-r'
```

Comment:
I think it is best not to add ```jgit``` as a new dependency to this project, and start relying on from now. It is best to extract this information directly from the local .git directory. JFrog CLI takes this approach already, and we cam just reimplement the code here.
I'd be happy to review the code again following this change.